warehouse/client/controlplane/client.go

```diff
@@ -46,6 +47,42 @@ type internalClient struct {
 	httpClient *http.Client
 }
 
+func (api *internalClient) GetSSHKeys(ctx context.Context, id string) (*PublicPrivateKeyPair, error) {
```

Comment:
Can we add test coverage for this?

Reply:
Let me see if I can add coverage for this in realistic time frame. Would it be possible to handle it as a separate PR in case the timelines exceed the cutoff time for the release ?

Reply:
Yes. I already approved the PR.